src/main/java/com/conveyal/datatools/manager/models/FeedSource.java

```diff
@@ -256,12 +256,24 @@ public URL getLatestUrl() {
      * Fetch the latest version of the feed.
      */
     public FeedVersion fetch (EventBus eventBus, String fetchUser) {
+        return fetch_url(eventBus, fetchUser, this.getLatestUrl());
+    }
+
+    public FeedVersion fetch_url (EventBus eventBus, String fetchUser, URL url) {
```

Comment:
Why do you leave it public, shouldn't be private/protected now ?

Reply:
yeah it can be, I'll change that.

Reply:
Done